tests/system/src/test/java/test/com/sun/javafx/image/impl/ImageRaceTest.java

```diff
@@ -125,8 +126,7 @@ static void init(String[] args) {
                 } else if (arg.equalsIgnoreCase("-verbose")) {
                     verbose = true;
                 } else {
-                    System.err.println("Unrecognized argument: "+arg);
-                    System.exit(-1);
+                    fail("Unrecognized argument: "+arg);
```

Comment:
maybe add spaces: "_+_arg

Reply:
Could be done indeed while this line is edited.